modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java

```diff
@@ -252,6 +260,14 @@ public NotificationQueueEntry updateSent(
 		return notificationQueueEntryPersistence.update(notificationQueueEntry);
 	}
 
+	public NotificationQueueEntry updateStatus(
```

Comment:
You don't need this method. Also, you can just call `notificationQueueEntry.persist()` to update the entry. ;)

Reply:
It was a valuable tip!
Besides, I forgot to run `gw buildService` after the method creation. Won't need it anymore.